hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -204,6 +206,16 @@ podman push --tls-verify=false "${SRIOV_NETWORK_OPERATOR_IMAGE}"
 podman push --tls-verify=false "${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE}"
 podman push --tls-verify=false "${SRIOV_NETWORK_WEBHOOK_IMAGE}"
 
+
```

Comment:
you also need to add in line 240 the export

Reply:
right!

Reply:
you didn't update here https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/518/files#diff-90e627cba9db282e04456e7e550bd478d2c12fbbbce4104f9421d640eb3a6447R247

Reply:
Image env variable is already exported here
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/518/files#diff-90e627cba9db282e04456e7e550bd478d2c12fbbbce4104f9421d640eb3a6447R212 

What need to be done on L247?